tests/h/auth/policy_test.py

```diff
@@ -143,14 +139,16 @@ def test_authenticated_userid_proxies_to_user_policy_first(self,
         assert client_policy.authenticated_userid.call_count == 0
         assert userid == user_policy.authenticated_userid.return_value
 
-    @pytest.mark.parametrize('path', AUTH_CLIENT_PATHS)
+    @pytest.mark.parametrize('route_name,route_method', AUTH_CLIENT_API_WHITELIST)
```

Comment:
All of these test changes are to account for the different way that auth-client routes are whitelisted and validated (no other functional changes).